### DISCUSSION TITLE
Real user feedback

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#54] [Q&A] [answered]
Did you receive any feedback from real users?

**Comment by Stagyrite** [comment, marked as answer]
Yes, I received an email when the project [started to compile again](https://github.com/Stagyrite/Perl-1.0/commit/a080e3707af43e140d5a8ae4e1b6424e99fcf1c6 "Make it compile again. · Stagyrite/Perl-1.0@a080e37"). At the point of stating the obvious, the whole development process was done in 1997. Nevertheless, the development continues, as Perl 1 doesn't compile without warnings. There has been [a social media post](https://twitter.com/effconia/status/1765400696537354351 "Maciej Matiaszowski on X: &quot;https://t.co/7Bo0uCaBKT&quot; / X") about this project, but I don't try to maintain any contact.
